config/runtime.exs

```diff
@@ -23,7 +22,8 @@ if config_env() != :test do
     message_log_zip_url: System.get_env("MESSAGE_LOG_ZIP_URL"),
     message_log_s3_bucket: System.get_env("MESSAGE_LOG_S3_BUCKET"),
     message_log_s3_folder: System.get_env("MESSAGE_LOG_S3_FOLDER"),
-    message_log_report_s3_folder: System.get_env("MESSAGE_LOG_REPORT_S3_FOLDER")
+    message_log_report_s3_folder: System.get_env("MESSAGE_LOG_REPORT_S3_FOLDER"),
+    s3_active_headend_path: "/headend.json"
```

Comment:
Now that this doesn't change at runtime, how about just putting it into a module constant in `ExternalConfig.S3`?

Reply:
Sure thing!